Custom nodes with no inputs in `io.Schema` no longer work

### Custom Node Testing

- [ ] I have tried disabling custom nodes and the issue persists (see [how to disable custom nodes](https://docs.comfy.org/troubleshooting/custom-node-issues#step-1%3A-test-with-all-custom-nodes-disabled) if you need help)

### Expected Behavior

`io.Schema` suggests that the `inputs` arguments is optional, so it should not result in errors if it is not supplied.

### Actual Behavior

TypeError: 'NoneType' object is not iterable

```
Traceback (most recent call last):
  File "C:\Dev\ComfyUI\server.py", line 683, in get_object_info
    out[x] = node_info(x)
             ^^^^^^^^^^^^
  File "C:\Dev\ComfyUI\server.py", line 645, in node_info
    return obj_class.GET_NODE_INFO_V1()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Dev\ComfyUI\comfy_api\latest\_io.py", line 1571, in GET_NODE_INFO_V1
    info = schema.get_v1_info(cls)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Dev\ComfyUI\comfy_api\latest\_io.py", line 1299, in get_v1_info
    input = create_input_dict_v1(self.inputs, live_inputs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Dev\ComfyUI\comfy_api\latest\_io.py", line 1384, in create_input_dict_v1
    add_to_input_dict_v1(input, inputs, live_inputs)
  File "C:\Dev\ComfyUI\comfy_api\latest\_io.py", line 1388, in add_to_input_dict_v1
    for i in inputs:
             ^^^^^^
TypeError: 'NoneType' object is not iterable
```

### Steps to Reproduce

Simple custom node which triggers this behavior:

```
from comfy_api.latest import ComfyExtension, io

class TestNode(io.ComfyNode):
    @classmethod
    def define_schema(cls):
        return io.Schema(
            node_id="TestNode",
            outputs=[io.Int.Output(display_name="foo")],
        )

    @classmethod
    def execute(cls, **kwargs):
        return io.NodeOutput(42)

class TestNodes(ComfyExtension):
    async def get_node_list(self) -> list[type[io.ComfyNode]]:
        return [TestNode]

async def comfy_entrypoint():
    return TestNodes()
```

Steps:
1. add the custom node
1. `python main.py`
1. load web ui in browser to trigger /object_info query

### Debug Logs

```powershell
Checkpoint files will always be loaded safely.
Total VRAM 12282 MB, total RAM 32694 MB
pytorch version: 2.8.0+cu128
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4070 : cudaMallocAsync
Using async weight offloading with 2 streams
Enabled pinned memory 14712.0
Using pytorch attention
Python version: 3.12.9 (main, Feb 12 2025, 14:52:31) [MSC v.1942 64 bit (AMD64)]
ComfyUI version: 0.3.76
ComfyUI frontend version: 1.33.10
[Prompt Server] web root: C:\Dev\ComfyUI\.venv\Lib\site-packages\comfyui_frontend_package\static
Total VRAM 12282 MB, total RAM 32694 MB
pytorch version: 2.8.0+cu128
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4070 : cudaMallocAsync
Using async weight offloading with 2 streams
Enabled pinned memory 14712.0

Import times for custom nodes:
   0.0 seconds: C:\Dev\ComfyUI\custom_nodes\Testo

Context impl SQLiteImpl.
Will assume non-transactional DDL.
No target revision found.
Starting server

To see the GUI go to: http://127.0.0.1:8188
[ERROR] An error occurred while retrieving information for the 'TestNode' node.
Traceback (most recent call last):
  File "C:\Dev\ComfyUI\server.py", line 683, in get_object_info
    out[x] = node_info(x)
             ^^^^^^^^^^^^
  File "C:\Dev\ComfyUI\server.py", line 645, in node_info
    return obj_class.GET_NODE_INFO_V1()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Dev\ComfyUI\comfy_api\latest\_io.py", line 1571, in GET_NODE_INFO_V1
    info = schema.get_v1_info(cls)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Dev\ComfyUI\comfy_api\latest\_io.py", line 1299, in get_v1_info
    input = create_input_dict_v1(self.inputs, live_inputs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Dev\ComfyUI\comfy_api\latest\_io.py", line 1384, in create_input_dict_v1
    add_to_input_dict_v1(input, inputs, live_inputs)
  File "C:\Dev\ComfyUI\comfy_api\latest\_io.py", line 1388, in add_to_input_dict_v1
    for i in inputs:
             ^^^^^^
TypeError: 'NoneType' object is not iterable
```

### Other

I believe this was introduced in #10832, it was not an issue in previous versions.

It's easy to workaround by passing `inputs=[]` to `io.Schema`, but IMO it's still a bug, and might also break other existing and future nodes.

--- a/comfy_api/latest/_io.py
+++ b/comfy_api/latest/_io.py
@@ -5,7 +5,7 @@ import inspect
 from abc import ABC, abstractmethod
 from collections import Counter
 from collections.abc import Iterable
-from dataclasses import asdict, dataclass
+from dataclasses import asdict, dataclass, field
 from enum import Enum
 from typing import Any, Callable, Literal, TypedDict, TypeVar, TYPE_CHECKING
 from typing_extensions import NotRequired, final
@@ -1199,9 +1199,9 @@ class Schema:
     """Display name of node."""
     category: str = "sd"
     """The category of the node, as per the "Add Node" menu."""
-    inputs: list[Input]=None
-    outputs: list[Output]=None
-    hidden: list[Hidden]=None
+    inputs: list[Input] = field(default_factory=list)
+    outputs: list[Output] = field(default_factory=list)
+    hidden: list[Hidden] = field(default_factory=list)
     description: str=""
     """Node description, shown as a tooltip when hovering over the node."""
     is_input_list: bool = False

